src/module/actor/ABFActorSheet.js

```diff
@@ -72,6 +72,24 @@ export default class ABFActorSheet extends ActorSheet {
     return 1000;
   }
 
+  render(force = false, options = {}) {    
+    const user = game.user;
+    const permissions = this.actor.ownership;
+    if (permissions[user.id] <= CONST.DOCUMENT_PERMISSION_LEVELS.LIMITED){
+      this.displayActorImagePopout();
+      return;
+    }  
+    return super.render(force, options);
+  }
```

Comment:
```suggestion
  /**
   * Tests if a given user has permission to render the ActorSheet.
   * If it does not, instead of rendering the sheet, shows the Actor's portrait.
   * @param {ABFActor} user
   * @returns {boolean}
   */
  _canUserView(user) {
    const canView = this.actor.testUserPermission(user, 'OBSERVER');
    if (!canView) {
      this.displayActorImagePopout();
    }
    return canView;
  }
```